src/langs/index.ts

```diff
@@ -0,0 +1,27 @@
+import translationsEN from './en/translations';
+import translationsFR from './fr/translations';
+import chatboxEN from './en/chatbox';
+import chatboxFR from './fr/chatbox';
+
+// define namespaces
+export const namespaces = {
+  translations: 'translations',
```

Comment:
Nice! I'm actually thinking about changing `translations` for `messages`? Since it's only messages currently.

Reply:
@pyphilia OK i will change it to `messages` everywhere